### DISCUSSION TITLE
Extract only source code necessary for json::parse(), dump(), .get<std::string>(), json::array(), .push_back()

### DISCUSSION THREAD

**Discussion by guest271314** [nlohmann/json#3401] [Q&A] [answered]
I asked this question over on https://www.reddit.com/r/cpp_questions/comments/tivjrt/how_to_identify_only_the_code_necessary_in_jsonhpp/i1gem8h/.

I'm wondering how to extract only the code necessary for `json::parse()`, `dump()`, `.get<std::string>()`, `json::array()`, `.push_back()`?

Thanks.

**Comment by nlohmann** [comment, marked as answer]
See https://www.reddit.com/r/cpp_questions/comments/tivjrt/comment/i1kaqjk/?utm_source=reddit&utm_medium=web2x&context=3.
